staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -53,7 +54,7 @@ func NewAdmissionOptions() *AdmissionOptions {
 	options := &AdmissionOptions{
 		Plugins:                &admission.Plugins{},
 		PluginNames:            []string{},
-		RecommendedPluginOrder: []string{lifecycle.PluginName},
+		RecommendedPluginOrder: []string{initialization.PluginName, lifecycle.PluginName},
```

Comment:
Initializers need to be `DefaultOff`.

Reply:
I see. I'll remove it.

Reply:
> I see. I'll remove it.

It is still in the recommended order.  There is a separate field `DefaultOff` which turns it off unless it is specifically requested.

Reply:
Thanks. Done.